src/main/java/io/openapitools/swagger/GenerateMojo.java

```diff
@@ -152,14 +156,34 @@ private Application resolveApplication(JaxRSScanner reflectiveScanner) {
         Class<? extends Application> appClazz = (Class<? extends Application>)clazz;
         return ClassUtils.createInstance(appClazz);
     }
-    
+
     private URLClassLoader createClassLoader() {
         try {
-            File compiled = new File(project.getBuild().getOutputDirectory());
```

Comment:
The compiled classes is part of `project.getCompileClasspathElements()`? I guess they must be if the integration tests is still running :-)

Reply:
I actually forgot this, but as a matter of fact, the `project.getCompileClasspathElements()` contains this. I would rather add this explicitly then to rely on the fact that it is actually contained in the compile dependencies.